roles/ssh_host_signer/defaults/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 ssh_host_signer_ca_keypair_dir: '/etc/ssh'
-ssh_host_signer_ca_private_key: "{{ ssh_host_signer_ca_keypair_dir }}/ca-key"
+ssh_host_signer_ca_private_key: "{{ ssh_host_signer_ca_keypair_dir }}/{{ stack_name }}-ca"
```

Comment:
`ssh_host_signer_ca_private_key: "{{ ssh_host_signer_ca_keypair_dir }}/{{ stack_name }}-ca"`
is already configured in `group_vars/all/vars.yml`. That value in `group_vars` will overrule the one from the role `defaults`. This makes it possible to re-use this role at some point in time in another repo, that does have a variable named `{{ stack_name }}`